src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
```

Comment:
```suggestion
  constructor(name, health = DEFAULT_HEALTH) {
    this.name = name;
    this.health = health;
    
    Animal.alive.push(this);
  }
```
- You should accept health in case it will be different, just by default it should be 100
- 100 and 50 are magic values, create constants for them
- Only Herbivore has a `hidden` property
<img width="837" alt="image" src="https://github.com/user-attachments/assets/610f6e79-3e0a-4a6f-b033-aab6544171c2">